### DISCUSSION TITLE
GRIDFINITY!

### DISCUSSION THREAD

**Discussion by aderusha** [aderusha/DDD-Printable-Wall-Control-System#41] [Show and tell]
I ran across the amazing [Gridfinity project by Zach Freidman](https://www.youtube.com/watch?v=ra_9zU-mnl8) and he makes a compelling case for his style of modular organization parts for use on horizontal surfaces.  I've taken that concept and ran it up the wall, [adding new centerpieces](https://github.com/aderusha/DDD-Printable-Wall-Control-System/tree/main/Centerpieces/Gridfinity) to provide Wall Control mounted baseplates in various arrangements.  This allows one to use the full Gridfinity project on a Wall Control pegboard.

![](https://i.imgur.com/SMNBfbh.jpeg)
![](https://i.imgur.com/wO4ji08.jpeg)
![](https://i.imgur.com/RCCZ1HU.jpeg)

**Comment by wrench797** [comment]
Do these work with the horizontal Wall Control panels as well?

**Reply by aderusha** [reply]
They do!  Use with the [locking spacers for horizontal wall control](https://github.com/aderusha/DDD-Printable-Wall-Control-System/tree/main/Centerpieces/Locking_spacer_for_horizontal_Wall_Control).

**Reply by wrench797** [reply]
Excellent. Thank you. 

I am having another problem now. I was able to easily download the center section I need AND the locking piece for horizontal panels. I can NOT for the life of me figure out how to download angle brackets. Everything else had a download button in the 3d view window but none of the angle brackets do. Being very new to github, I am assuming it is something I am doing wrong. Please advise.

**Reply by aderusha** [reply]
Github sometimes has a hard time differentiating filetypes and can be kinda weird about downloads.  This link will grab the current state of the repo and everything in it:  https://github.com/aderusha/DDD-Printable-Wall-Control-System/archive/refs/heads/main.zip

**Comment by jamesalmeida** [comment]
Thanks for sharing!

**Comment by bdaneshgar** [comment]
This is great! Do you press fit magnets? What size magnets do you use. Thank you!

**Reply by aderusha** [reply]
Hey glad it's useful!  This was designed to use the same 6mm round by 2mm thick disc magnets used by the main gridfinity project.  Check details here: https://gridfinity.xyz/specification/

**Comment by ebroder** [comment]
I've been having great luck with this so far. I'd love to be able to setup a Gridfinity array that spans a full Wall Control panel (so something like the [7x14 Gridfinity Frame 4x8 model](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Gridfinity/7x14%20Gridfinity%20Frame%204x8.stl) but my printer isn't big enough to print that out.

Is it possible to do that with the models that exist today? I'm pretty sure it's not, so I'm going to take a stab at modeling something, but would love to know if I'm missing something about the system.

(I think I need to cut the 7x14/4x8 model in half and figure out some way to join it together at the center. I also think I need a variant on the mid-span support that is evenly spaced around the hook, instead of only protruding to one side, so that it can support each half of the centerpiece)
